### ISSUE TITLE
Crash with any attempt to use a removed Physics Object

### ISSUE THREAD

**Issue by NeatNit** [Facepunch/garrysmod-issues#1827]
Trying to print or otherwise use a physics object that has been removed will result in a crash. Tested in both live and dev.

My shitty code for testing it (put it in init.lua):
```lua
local phys = false

hook.Add("PlayerSay", "asjdkh", function(ply)
	if phys then print(phys:GetMass()) end
	
	tr = ply:GetEyeTrace()
	
	phys = tr.Entity:GetPhysicsObjectNum(tr.PhysicsBone)
end )
```

(You will need to say something while looking at a prop, then remove that prop, then say something else. Make sure you give the prop enough time to actually get removed, not just visually)

Also crashes without `:GetMass()`.

**Comment by Bo98** [comment]
This is fixed now (15.07.02)?

**Comment by robotboy655** [comment]
Yeah, it should be fixed some updates ago.
